ssas/groups.go

```diff
@@ -28,16 +29,38 @@ type Group struct {
 	Data    postgres.Jsonb `json:"data"`
 }
 
+func CreateGroup(gd GroupData) (Group, error) {
+	gdBytes, err := json.Marshal(gd)
+	if err != nil {
+		return Group{}, err
```

Comment:
Is this a function that needs `ssas.OperationStarted()` and `ssas.OperationSucceeded()`/`ssas.OperationFailed()`?  If so, we should start the logging before our first return.